content/docs/index.md

```diff
@@ -29,13 +27,17 @@ last_edited: '2021-07-27T15:51:56.737Z'
 
 The **TinaCMS Toolkit** empowers developers to build a visual editing experience directly on their sites. By creating a custom editing experience with Tina instead of opting for a conventional CMS, developers can give their teams a contextual, intuitive editing experience without sacrificing code control.
 
+[TinaCMS CLI](/docs/cli/) - The TinaCMS CLI can be used to set up your project with TinaCMS schema configuration, and run a local version of the TinaCMS API (using your file system's content).
```

Comment:
I think this is `/docs/cli-overview` now

Reply:
I can confirm it is not. it is just the base url